scripts/geo_metadata.py

```diff
@@ -188,35 +195,35 @@ def gather_objects(input_object, start_type=None):
 	return objs
 
 
-# Gather raw matrices by object type and 'background_barcodes_included' to select for filtered matrix from CR output
 def gather_rawmatrices(dataset):
+	"""
+	Gather raw matrices by object type associated with dataset with following filtering:
+	 - 'background_barcodes_included' to select for filtered matrix from CR output
+	 - 'status' = 'in progress' to select for valid objects
+	"""
 	my_raw_matrices = []
 	unfiltered_matrices = []
-	raw_ids = []
-	query_url = urljoin(server, 'search/?type=RawMatrixFile&dataset=/datasets/' + dataset + '/&format=json' + '&limit=all')
-	r = requests.get(query_url, auth=connection.auth)
-	try:
-		r.raise_for_status()
-	except requests.HTTPError:
-		sys.exit("Error in getting raw matrix files: {}".format(query_url))
-	else:
-		for raw in r.json()['@graph']:
-			raw_ids.append(raw['@id'])
-	for identifier in raw_ids:
-		obj = lattice.get_object(identifier, connection)
-		if obj['@type'][0] == 'RawMatrixFile':
-			if not obj.get('background_barcodes_included'):
-				my_raw_matrices.append(obj)
-			else:
-				unfiltered_matrices.append(obj)
-		else:
-			continue
-	return my_raw_matrices, unfiltered_matrices
 
-
-def get_dataset(dataset):
-	global geo_study
-	results = lattice.get_object(dataset, connection)
+	field_lst = RAW_MATRIX+['@id','accession', 'libraries', 'derived_from', 's3_uri', 'status']
```

Comment:
add md5sum here?

Reply:
Yes! That should be it. Thanks!

Reply:
Just tested it, and that was it. I'll push a commit to fix. Thanks!